ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -349,6 +349,9 @@ objects:
                         --key /tmp/artifacts/installer/tls/journal-gatewayd.key \
                         --url "https://${bootstrap_ip}:19531/entries?_SYSTEMD_UNIT=${service}.service"
                 done
+                ssh-add /tmp/cluster/ssh-privatekey
+                ssh -A core@${bootstrap_ip} /usr/local/bin/installer-gather.sh
```

Comment:
Can you do this in 1 command with `-i /tmp/cluster/ssh-privatekey` added to this command (and the one below it)?

Reply:
No, that doesn't appear to forward the credential properly.

Reply:
Interesting.